src/pages/library/test-datasets.ts

```diff
@@ -0,0 +1,60 @@
+import { Dataset } from 'src/libs/ajax/Catalog'
+
+
+export const TEST_DATASET_ONE: Dataset = {
+  'dct:title': 'The worlds best dataset',
+  'dct:description': 'The best there ever was',
+  'dct:creator': 'The hero we all need',
+  'dct:issued': new Date().toDateString(),
+  'dcat:accessURL': 'https://http.cat/#workspaces/',
```

Comment:
This has /#workspaces/ in order to register as a granted dataset, rather than external